.github/workflows/ci-unit-tests.yaml

```diff
@@ -13,7 +13,6 @@ jobs:
           - windows-latest
           - macos-latest
         node:
-          - 8
```

Comment:
Not removing 10 as well?

Reply:
I'll remove that when we get the policy into this repo